src/Menu/Menu.js

```diff
@@ -76,11 +75,7 @@ type Props = DefaultProps & {
   transitionDuration?: number | 'auto',
 };
 
-export const styleSheet = createStyleSheet('MuiMenu', {
-  root: {
-    maxHeight: 250,
-  },
-});
+export const styleSheet = createStyleSheet('MuiMenu', {});
```

Comment:
If entered is a dead key, then we don't need it. I'll double check. Otherwise withStyles requires a non null styleSheet to allow nested overrides.